### ISSUE TITLE
Issue with input and output names

### ISSUE THREAD

**Issue by rahulrs2320** [mdangschat/ctc-asr#17]
Hello, I am currently trying to freeze the graph from this model and I am unable to do so because when I am inspecting the "graph.pbtxt" created after training, there is no node with the name of "logits/dense". 

Please help me figure out what the output node name is so I can freeze the graph to .pb. 

Thank you
Regard
Rahul B
